### PR TITLE
ART-8115 Modify prepare-release job for prereleases

### PR DESCRIPTION
**Pull request by thegreyd**
https://issues.redhat.com/browse/ART-8115

Summary
- Add and use `--permissive` mode to `elliott find-bugs:sweep` to avoid erroring out on invalid bugs when we prep ECs and RCs.
- Build and attach bundle builds to prerelease advisories
- Skip attach-cve-flaws when we're prepping a prerelease assembly
- Gather dependencies from other advisories when prepping prerelease advisory
- Use `rhose async auto` release for preGA advisories - with https://github.com/openshift-eng/ocp-build-data/pull/4101
- Bring all prep-release slack messages in a single thread

### CODE REVIEW

File: pyartcd/pyartcd/pipelines/prepare_release.py [hunk 1]
```diff
@@ -195,21 +195,26 @@ async def run(self):
...
-        self.sweep_bugs()
+        if assembly_type in (AssemblyTypes.PREVIEW, AssemblyTypes.CANDIDATE):
+            self.sweep_bugs(permissive=True)
+        else:
+            self.sweep_bugs()
 
-        _LOGGER.info("Processing attached Security Trackers")
-        for _, advisory in advisories.items():
-            self.attach_cve_flaws(advisory)
+        if 'prerelease' in advisories:
```
Comment (joepvd) [inline pyartcd/pyartcd/pipelines/prepare_release.py:212]: Thinking that `assembly_type in (AssemblyTypes.PREVIEW, AssemblyTypes.CANDIDATE)` should skip flaw bug processing. There will be RCs that do not have a prerelease advisory, and I think we want to postpone CVE stuff until we're close to GA. 

Or is there a `software_lifecyckle.phase` that would match with the desired here?

Comment (thegreyd) [reply pyartcd/pyartcd/pipelines/prepare_release.py:212]: Yeah that was one of the things I wasn't sure about from the doc if Luke meant us to run cve stuff intermittently or all at end. It makes sense to run all at end, while still reporting on errors with --permissive. I'll make the change

Comment (thegreyd) [reply pyartcd/pyartcd/pipelines/prepare_release.py]: Done

**Review by joepvd — COMMENTED** [review]
/approve

**Pull request merged**